GIT_CONFLICTS.md

```diff
@@ -0,0 +1,77 @@
+# Wanderlust Project
+
+## Overview
+Welcome to the Wanderlust project! This is a collaborative effort to build an amazing application. We appreciate your contributions and aim to make the process as smooth as possible.
+
+## Common QnA for Git and Open Source Contributions
+
+<details>
+<summary>
+
+### How can I know if I’m in a bad situation?
+</summary>
+I suggest you to run  **`gitk`**  or  **`gitk --all`**. The commit tree should be self-explainatory.
+

```

Comment:
For drop down please refer the images section of this PR https://github.com/krishnaacharyaa/wanderlust/pull/67 and replicate

Reply:
I have created drop-down, I didn't find the raw code in that refrence but I have added. Please review.